appserver/orb/orb-iiop/src/main/java/org/glassfish/enterprise/iiop/impl/IIOPSSLSocketFactory.java

```diff
@@ -184,17 +169,13 @@ public IIOPSSLSocketFactory() {
                     if (outboundSsl != null) {
                         boolean ssl2Enabled = Boolean.valueOf(outboundSsl.getSsl2Enabled());
```

Comment:
I assume this should also cover ssl2? @Pandrex247

Reply:
Definitely - that's ancient 😂
Didn't realise that was even an option - it's not something you can select in the Admin Console